WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsView.swift

```diff
@@ -35,33 +37,44 @@ struct PaymentMethodsView: View {
 
             Divider()
 
-            Group {
-                MethodRow(icon: .priceImage, title: Localization.cash) {
-                    showingCashAlert = true
-                    viewModel.trackCollectByCash()
-                }
+            VStack(alignment: .leading, spacing: 8) {
+                VStack(alignment: .leading, spacing: Layout.noSpacing) {
+                    MethodRow(icon: .priceImage, title: Localization.cash) {
+                        showingCashAlert = true
+                        viewModel.trackCollectByCash()
+                    }
 
-                if viewModel.showPayWithCardRow {
-                    Divider()
+                    if viewModel.showPayWithCardRow {
+                        Divider()
 
-                    MethodRow(icon: .creditCardImage, title: Localization.card) {
-                        viewModel.collectPayment(on: rootViewController, onSuccess: dismiss)
+                        MethodRow(icon: .creditCardImage, title: Localization.card) {
+                            viewModel.collectPayment(on: rootViewController, onSuccess: dismiss)
+                        }
                     }
-                }
 
-                if viewModel.showPaymentLinkRow {
-                    Divider()
+                    if viewModel.showPaymentLinkRow {
+                        Divider()
 
-                    MethodRow(icon: .linkImage, title: Localization.link) {
-                        sharingPaymentLink = true
-                        viewModel.trackCollectByPaymentLink()
+                        MethodRow(icon: .linkImage, title: Localization.link) {
+                            sharingPaymentLink = true
+                            viewModel.trackCollectByPaymentLink()
+                        }
                     }
                 }
+                .padding(.horizontal)
+                .background(Color(.listForeground))
+
+                if viewModel.showUpsellCardReaderFeatureBanner {
+                    FeatureAnnouncementView(title: Localization.featureAnnouncementTitle,
+                                            message: Localization.featureAnnouncementMessage,
+                                            buttonTitle: Localization.featureAnnouncementButtonTitle,
```

Comment:
Should we be more specific with these constant names? For example:

- `featureAnnouncementTitle` -> `purchaseCardReaderAnnouncementTitle`
- `featureAnnouncementMessage` -> `purchaseCardReaderAnnouncementMessage`
- `featureAnnouncementButtonTitle` -> `purchaseCardReaderAnnouncementButtonTitle`